cfgov/regulations3k/regdown.py

```diff
@@ -218,6 +218,7 @@ def run(self, parent, blocks):
             else:
                 el = util.etree.SubElement(parent, 'p')
             el.set('id', label)
+            el.set('data-label', label)
```

Comment:
FWIW it probably would have been more performant to do the label parsing in python and pass the generated paragraph labels to the DOM with this data attribute. There are parallels between the regdown logic and the JS logic in this PR. Not a big deal, though.